app/controllers/admin/routes.py

```diff
@@ -272,9 +273,13 @@ def addRecurringEvents():
 @admin_bp.route('/userProfile', methods=['POST'])
 def userProfile():
     volunteerName= request.form.copy()
-    username = volunteerName['searchStudentsInput'].strip("()")
-    user=username.split('(')[-1]
-    return redirect(url_for('main.viewUsersProfile', username=user))
+    if volunteerName['searchStudentsInput']:
+        username = volunteerName['searchStudentsInput'].strip("()")
+        user=username.split('(')[-1]
+        return redirect(url_for('main.viewUsersProfile', username=user))
+    else:
+        flash(f"You have not entered anything.", category='danger')
```

Comment:
I think it would be better to have an error message that is a bit more descriptive. Maybe something along the lines of "Please enter a students first name or username to search."